### DISCUSSION TITLE
When am I expecting to receive my Academic Benefits?

### DISCUSSION THREAD

**Discussion by alexpanrui** [community/community#141849] [GitHub Education] [answered]
### Hi,

Hi, today is the 5th day since my application being approved, yet I still don't receive the academic benefits. Please help.

**Comment by marlonwq** [comment]
GitHub usually takes a while to make its benefits available, when it is available it will look purple like this:

!['](https://github.com/user-attachments/assets/6330c979-e964-4eae-b22f-4ff12b6f48f1)
For me it took about 7-8 days.

**Reply by alexpanrui** [reply]
Thx, man.

**Reply by marlonwq** [reply, marked as answer]
I think it must have already been approved, try taking a look again.

**Reply by alexpanrui** [reply]
Yah, I've got it. Thx.
